api-tests/src/test/java/org/openmrs/module/reporting/report/definition/service/ReportDefinitionServiceImplTest.java

```diff
@@ -39,11 +42,33 @@
 	@Test
 	public void purgeDefinition_shouldPurgeReportDesigns() throws Exception {
 		executeDataSet("org/openmrs/module/reporting/include/ReportDefinitionServiceImplTest.xml");
-    	
+

```

Comment:
I know this wasn't done originally, but can you add in checks here to confirm that report designs 3 and 4 are non null before the purge definition operation?